forge-gui/res/cardsfolder/upcoming/pirated_copy.txt

```diff
@@ -0,0 +1,9 @@
+Name:Pirated Copy
+ManaCost:4 U
+Types:Creature Shapeshifter Pirate
+PT:0/0
+K:ETBReplacement:Copy:DBCopy:Optional
+SVar:DBCopy:DB$ Clone | Choices$ Creature.Other | AddTypes$ Pirate | AddTriggers$ DrawTrig | AddSVars$ DrawTrig,TrigDraw | SpellDescription$ You may have CARDNAME enter the battlefield as a copy of any creature on the battlefield, except it's a Pirate in addition to its other types and it has "Whenever this creature or another creature with the same name deals combat damage to a player, you draw a card."
```

Comment:
try without `AddSVars` - you definitely shouldn't need to add svar DrawTrig and probably TrigDraw is automatically added too